llvm/test/tools/llvm-objcopy/COFF/subsystem.test

```diff
@@ -30,6 +30,15 @@
 # INVALID-MAJOR-NUMBER: 'foo' is not a valid subsystem major version
 # INVALID-MINOR-NUMBER: 'bar' is not a valid subsystem minor version
 
+# RUN: llvm-objcopy --subsystem=efi_application %t.in.exe
+# RUN: llvm-objcopy --subsystem=efi-app %t.in.exe
+
+# RUN: llvm-objcopy --subsystem=efi_boot_service_driver %t.in.exe
+# RUN: llvm-objcopy --subsystem=efi-bsd %t.in.exe
+
+# RUN: llvm-objcopy --subsystem=efi_runtime_driver %t.in.exe
+# RUN: llvm-objcopy --subsystem=efi-rtd %t.in.exe
```

Comment:
I think you need to use llvm-readobj/FileCheck after each call to check the output file. Otherwise, this does nothing other than check that llvm-objcopy understands the value (but it might do absolutely nothing with it, for example). Once you do that, you should probably use a separate output file, so that you're not modifying the input in place, messing it up for other executions in the same test.

Reply:
Thanks. I've amended the test to check for the actual output too.